hugr-py/docs/api-docs/conf.py

```diff
@@ -36,6 +37,18 @@
 
 templates_path = ["_templates"]
 exclude_patterns = ["_build", "Thumbs.db", ".DS_Store", "conftest.py"]
+html_sidebars = {
+    "**": [
+        "navbar-logo.html",
+        "icon-links.html",
+        "search-button-field.html",
+        "sbt-sidebar-nav.html",
```

Comment:
Are these all available by default?

Reply:
yeah they're included with the theme, they're the default ones I just had to re-add them because I was customising